File: examples/2.high-level/python/05.import/01.pip/01.sys/program1.py
# https://www.askpython.com/python/python-command-line-arguments

import sys

sentence = sys.argv[1]

def count_words(sentence):
    summary = {}
    words = sentence.split(' ')
    for word in words:
        if word in summary:
            summary[word] += 1
        else:
            summary[word] = 1
    return summary

if __name__ == '__main__':
    text = sys.argv[1] # Error?
    palabras = count_words(sentence)

for palabra, cantidad in palabras.items():
    print(f'{palabra}: {cantidad}')

File: examples/2.high-level/python/05.import/01.pip/01.sys/program1.rb
#!/usr/bin/ruby

def count_words(sentence)
  summary = Hash.new(0)
  words = sentence.split(' ')
  words.each { |word| summary[word] += 1 }
  summary
end

sentence = ARGV[0]
palabras = count_words(sentence)
palabras.each_pair { |key,value| puts "#{key}: #{value}" }
